### DISCUSSION TITLE
OneNote for Windows 10

### DISCUSSION THREAD

**Discussion by jelmer1** [elvirbrk/NoteHighlight2016#173] [Q&A]
Does this work with OneNote for Windows 10?
![onenote-version](https://user-images.githubusercontent.com/110474639/182391148-abb637ab-be92-4399-b79e-41064a986050.PNG)

**Comment by ArnoldWillems** [comment]
Hi @jelmer1,

I just found this add-in and installed it on the same version as you refer to, but at my side it's not working.
It would be  a great add-in to get working on this version as I use OnenOte a lot to store my ideas in.
